splicemachine/ml/management.py

```diff
@@ -37,11 +169,100 @@ def __str__(self):
     @staticmethod
     def __removekey(d, key):
         """
-        Remove a key from a dictionary
+        return self.active_run.info.run_uuid
+
+    @property
+    def experiment_id(self):
+        """
+        Returns the UUID of the current experiment
+        """
+        return self.active_experiment.experiment_id
+
+    def __repr__(self):
+        """
+        Return String Representation of Current MLManager
+        :return:
+        """
+        return "MLManager\n" + ('-' * 20) + "\nCurrent Experiment: " + \
+               str(self.active_experiment) + "\n"
+
+    def __str__(self):
+        return self.__repr__()
+
+    def check_active(func):
+        """
+        Decorator to make sure that run/experiment
+        is active
+        """
+
+        def wrapped(self, *args, **kwargs):
+            if not self.active_experiment:
+                raise Exception("Please either use set_active_experiment or create_experiment "
+                                "to set an active experiment before running this function")
+            elif not self.active_run:
+                raise Exception("Please either use set_active_run or create_run to set an active "
+                                "run before running this function")
+            else:
+                return func(self, *args, **kwargs)
+
+        return wrapped
+
+    def lp(self, *args, **kwargs):
+        """
+        Shortcut function for logging
+        parameters
+        """
+        return self.log_param(*args, **kwargs)
+
+    def lm(self, *args, **kwargs):
+        """
+        Shortcut function for logging
+        metrics
         """
-        r = dict(d)
-        del r[key]
-        return r
+        return self.log_metric(*args, **kwargs)
+
+    def st(self, *args, **kwargs):
+        """
+        Shortcut function for setting tags
+        """
+        return self.set_tag(*args, **kwargs)
+
+    def create_experiment(self, experiment_name, reset=False):
+        """
+        Create a new experiment. If the experiment
+        already exists, it will be set to active experiment.
+        If the experiment doesn't exist, it will be created
+        and set to active. If the reset option is set to true
+        (please use with caution), the runs within the existing
+        experiment will be deleted
+        :param experiment_name: (str) the name of the experiment to create
+        :param reset: (bool) whether or not to overwrite the existing run
+        """
+        experiment = self.get_experiment_by_name(experiment_name)
+        if experiment and experiment.lifecycle_stage == 'active':
+            print(
```

Comment:
This is a comment overall of this file.  Do we want to use logging instead of printing to the console.  It looks like further down there are methods for logging properly

Reply:
In some cases, yes. But many of these prints are for the user as well. But I should go back and add logging. Maybe after merge, make a new Jira. Or before merge, worth discussing.